src/meson.build

```diff
@@ -0,0 +1,17 @@
+raylib_src = [
+  'raudio.c',
+  'rcore.c',
+  'rglfw.c',
+  'rmodels.c',
+  'rshapes.c',
+  'rtext.c',
+  'rtextures.c',
+  'utiles.c'
+]
+
+ray_lib = static_library('raylib,
```

Comment:
This should be marked for installation (`install: true`). It also should specify the

```meson
version: '4.0.0',
soversion: '400',
```

Reply:
Yea that makes sense, gonna add that plus some of the helpful suggestions to the first draft.